src/components/BlipGroupCard.vue

```diff
@@ -165,23 +159,26 @@ export default {
         status: this.documents[0].status
       }
       for (let i = 1; i < this.documents.length; i++) {
-        const message = this.documents[i]
-        if (this.compareMessages(group.msgs[group.msgs.length - 1], message)) {
-          group.msgs.push(message)
-          group.date = message.date
-          group.status = message.status
-          group.reason = message.reason
+        const message1 = group.msgs[group.msgs.length - 1]
+        const message2 = this.documents[i]
```

Comment:
Is it possible to change this message1 / 2 a better name? like currentMessage, lastMessage?

Reply:
It is! I was just following the names given in the 'compareMessages' method. I'll change them right away.

Reply:
Done! I changed the group.msgs[group.msgs.length - 1] to lastMessage to reflect its position, and the this.documents[i] to currentMessage to indicate that it is the one we are currently iterating.